examples/Wall/dodo_wall.py

```diff
@@ -22,47 +22,54 @@
 
 # define required parameters
 params = {  # geometry parameters
-    "length": 200,  # mm
-    "height": 200,  # mm
-    "width": 200,  # mm
+    "length": 150,  # mm
+    "height": 150,  # mm
+    "width": 150,  # mm
     "radius": None,  # mm
-    "infill": "solid",
+    "infill": "zigzag",
```

Comment:
We should think about how we can run this in our git actions for different infill parameters
That we are aware if we break something

Reply:
I would look into it